src/test/java/spoon/test/prettyprinter/SniperAnnotatedEnumTest.java

```diff
@@ -0,0 +1,59 @@
+package spoon.test.prettyprinter;
+
+import org.apache.commons.io.*;
+import org.hamcrest.*;
+import org.junit.*;
+import static org.junit.Assert.*;
```

Comment:
You should avoid using JUnit 4. The `assertThat` method should be imported from `org.hamcrest.MatcherAssert` instead.